diff --git a/src/procedure/DELETE_CHAR.sql b/src/procedure/DELETE_CHAR.sql
index e34bcb9..59b29c6 100644
--- a/src/procedure/DELETE_CHAR.sql
+++ b/src/procedure/DELETE_CHAR.sql
@@ -61,9 +61,8 @@ DECLARE
   @PX int,
   @PZ int,
   @PY int,
-  @strSkill varchar(20),
-  @strItem binary(400)
-DECLARE @Members smallint
+  @dwTime int
+DECLARE @bySkill binary(10), @byItem binary(400), @bySerial binary(400), @Members smallint
 
 BEGIN TRAN
 
@@ -135,8 +134,10 @@ SELECT
   @PX = PX,
   @PZ = PZ,
   @PY = PY,
-  @strSkill = strSkill,
-  @strItem = strItem
+  @dwTime = dwTime,
+  @bySkill = bySkill,
+  @byItem = byItem,
+  @bySerial = bySerial
 FROM USERDATA WHERE strUserId = @CharID
 
 INSERT INTO DELETED_USERDATA (
@@ -171,8 +172,10 @@ INSERT INTO DELETED_USERDATA (
   PX,
   PZ,
   PY,
-  strSkill,
-  strItem
+  dwTime,
+  bySkill,
+  byItem,
+  bySerial
 )
 VALUES (
   @AccountID,
@@ -206,8 +209,10 @@ VALUES (
   @PX,
   @PZ,
   @PY,
-  @strSkill,
-  @strItem
+  @dwTime,
+  @bySkill,
+  @byItem,
+  @bySerial
 )
 
 DELETE FROM USERDATA WHERE strUserId = @CharID
