diff --git a/src/schema/WAREHOUSE.sql b/src/schema/WAREHOUSE.sql
index ae2ae02..63a77d4 100644
--- a/src/schema/WAREHOUSE.sql
+++ b/src/schema/WAREHOUSE.sql
@@ -6,8 +6,8 @@ CREATE TABLE [dbo].[WAREHOUSE](
 	[strAccountID] [char](21) NOT NULL,
 	[nMoney] [int] NOT NULL,
 	[dwTime] [int] NOT NULL,
-	[WarehouseData] [char](1600) NULL,
-	[strSerial] [char](20) NULL,
+	[byItem] [binary](1600) NULL,
+	[bySerial] [binary](1600) NULL,
  CONSTRAINT [PK_WAREHOUSE] PRIMARY KEY CLUSTERED 
 (
 	[strAccountID] ASC
