diff --git a/src/procedure/LOAD_USER_DATA.sql b/src/procedure/LOAD_USER_DATA.sql
index b40b666..4a33232 100644
--- a/src/procedure/LOAD_USER_DATA.sql
+++ b/src/procedure/LOAD_USER_DATA.sql
@@ -5,52 +5,53 @@ GO
 
 
 CREATE PROCEDURE [dbo].[LOAD_USER_DATA]
-  @id char(21)
+  @id char(21),
+  @nRet smallint OUTPUT
 AS
 
-DECLARE @nRet smallint
 SET @nRet = 0
 
 SELECT @nRet = COUNT(strUserId) FROM USERDATA WHERE strUserId = @id
 IF @nRet = 0
-  RETURN 0
+  RETURN
 
-SELECT
-  Nation,
-  Race,
-  [Class],
-  HairColor,
-  Rank,
-  Title,
-  [Level],
-  [Exp],
-  Loyalty,
-  Face,
-  City,
-  Knights,
-  Fame,
-  Hp,
-  Mp,
-  Sp,
-  Strong,
-  Sta,
-  Dex,
-  Intel,
-  Cha,
-  Authority,
-  Points,
-  Gold,
-  [Zone],
-  Bind,
-  PX,
-  PZ,
-  PY,
-  dwTime,
-  strSkill,
-  strItem,
-  strSerial
-FROM USERDATA WHERE strUserId = @id
+  SELECT
+    Nation,
+    Race,
+    [Class],
+    HairColor,
+    Rank,
+    Title,
+    [Level],
+    [Exp],
+    Loyalty,
+    Face,
+    City,
+    Knights,
+    Fame,
+    Hp,
+    Mp,
+    Sp,
+    Strong,
+    Sta,
+    Dex,
+    Intel,
+    Cha,
+    Authority,
+    Points,
+    Gold,
+    [Zone],
+    Bind,
+    PX,
+    PZ,
+    PY,
+    dwTime,
+    strSkill,
+    strItem,
+    strSerial
+  FROM USERDATA WHERE strUserId = @id
 
-RETURN 1
+SET @nRet = 1
+RETURN
 
 GO
