diff --git a/src/procedure/ACCOUNT_LOGIN.sql b/src/procedure/ACCOUNT_LOGIN.sql
index aadf75a..04960ef 100644
--- a/src/procedure/ACCOUNT_LOGIN.sql
+++ b/src/procedure/ACCOUNT_LOGIN.sql
@@ -9,11 +9,6 @@ CREATE PROCEDURE [dbo].[ACCOUNT_LOGIN]
 
 AS
 
-
-DECLARE @Nation tinyint, @CharNum smallint
-SET @Nation = 0
-SET @CharNum = 0
-
 DECLARE @pwd varchar(13)
 
 SET @pwd = NULL
@@ -21,34 +16,16 @@ SET @pwd = NULL
 SELECT @pwd = strPasswd FROM [dbo].[TB_USER] WHERE strAccountID = @AccountID
 IF @pwd IS NULL
   BEGIN
-    SET @nRet = 0
+    SET @nRet = 2
     RETURN
   END
 
 ELSE IF @pwd != @Password
   BEGIN
-    SET @nRet = 0
+    SET @nRet = 3
     RETURN
   END
 
-SELECT
-  @Nation = bNation,
-  @CharNum = bCharNum
-FROM ACCOUNT_CHAR WHERE strAccountID = @AccountID
-IF @@ROWCOUNT = 0
-  BEGIN
-    SET @nRet = 1
-    RETURN
-  END
-IF @CharNum = 0
-  BEGIN
-    SET @nRet = 1
-    RETURN
-  END
-ELSE
-  BEGIN
-    SET @nRet = @Nation + 1
-    RETURN
-  END
+SET @nRet = 1 -- success
 
 GO
